loader/include/Geode/loader/Mod.hpp

```diff
@@ -163,17 +168,26 @@ namespace geode {
         /**
          * Get a mod-specific launch argument. This is equivalent to `Loader::getLaunchArgument`
          * with the argument name prefixed by the mod ID. For example, a launch argument named
-         * `modArg` for the mod `author.test` would be specified with `--geode:author.test.modArg=value`.
+         * `mod-arg` for the mod `author.test` would be specified with `--geode:author.test.mod-arg=value`.
          * @param name The argument name
          * @return The value, if present
          */
         std::optional<std::string> getLaunchArgument(std::string_view const name) const;
         /**
-         * Equivalent to a prefixed `Loader::getLaunchBool` call. See `Mod::getLaunchArgument`
+         * Equivalent to a prefixed `Loader::getLaunchFlag` call. See `Mod::getLaunchArgument`
          * for details about mod-specific launch arguments.
          * @param name The argument name
          */
-        bool getLaunchBool(std::string_view const name) const;
+        bool getLaunchFlag(std::string_view const name) const;
```

Comment:
same here

Reply:
this is fine because no mod uses this yet